app/views/shared/_attributes.html.erb

```diff
@@ -1,8 +1,11 @@
-      <table class="table">
+<% include_title ||= false # title is part of the exterior template in common cases %>
```

Comment:
I feel like the ticketing set us up here.  These should have been the same ticket.